controller/src/controller.rs

```diff
@@ -205,6 +264,11 @@ impl<T: BottlerocketShadowClient> BrupopController<T> {
                 }
             }
 
+            // Cleanup BRS when the operator is removed from a node
```

Comment:
I was wondering if it would be better to add more explanation here to illustrate that even the above steps moved a invalid Brs to active, the clean up would catch later here.